lib/internal/Magento/Framework/Image/Adapter/ImageMagick.php

```diff
@@ -333,7 +345,11 @@ public function watermark($imagePath, $positionX = 0, $positionY = 0, $opacity =
                 );
             }
         } catch (\ImagickException $e) {
-            throw new \Exception('Unable to create watermark.', $e->getCode(), $e);
+            throw new LocalizedException(
+                new \Magento\Framework\Phrase('Unable to create watermark.'),
```

Comment:
`__('Unable to create watermark.')`

Reply:
I think the whole idea behind LocalizedException and Phrase is not to use translation function on top of it.

Reply:
Nowhere in code it is used this way

Reply:
`__()` actually is new Phrase - take a look into https://github.com/magento/magento2/blob/c7c0a992d83de56f15555d3edcecef7aaa32f0d7/app/functions.php#L19

Reply:
So using new Phrase is cleaner (OOP not superglobal function)

Reply:
Both ways are correct unless you use full path for all `Phrase`s :-)

Reply:
OK i checked other places in Magento. Changed